### PR TITLE
Removes dynamic dimensions for V2 Skeletons in favor of classnames

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

- Adds deprecation notices to V1 Skeletons
- Removes `width` and `height` props from V2 Skeletons
- Migrates V2 skeletons to use classnames

For more context on this decision: https://tailwindcss.com/docs/content-configuration#dynamic-class-names 

## Type of change

- [x] Chore (refactoring code, technical debt, workflow improvements)

### CODE REVIEW

**Review by zomars — COMMENTED** [review]
Ready for review 

Motion Capture Dance GIF by Ankita

**Review by sean-brydon — APPROVED** [review]
LGTM - whoever wrote them dynamic classNames should be shot 😅 (It was me)

File: packages/ui/skeleton/index.tsx [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
   height?: string;
   className?: string;
 };
 
 interface AvatarProps extends SkeletonBaseProps {
-  // Limit this cause we don't use avatars bigger than thi
-  width: "2" | "3" | "4" | "5" | "6" | "8" | "12";
-  height: "2" | "3" | "4" | "5" | "6" | "8" | "12";
+  // Limit this cause we don't use avatars bigger than this
+  /** @deprecated @see https://tailwindcss.com/docs/content-configuration#dynamic-class-names */
```
Comment (PeerRich) [inline packages/ui/skeleton/index.tsx:13]: why show deprecated when we just replace it everywhere?

**Review by PeerRich — APPROVED** [review]
does it change anything visually? or just a refactor cleanup?

File: packages/ui/skeleton/index.tsx (hunk 1 above)
Comment (zomars) [reply packages/ui/skeleton/index.tsx:13]: Not worth the effort now as we will remove it post v2

**Pull request merged**